client/my-sites/stats/stats-views/months.jsx

```diff
@@ -87,42 +83,33 @@ const StatsViewsMonths = ( props ) => {
 		return sum;
 	};
 
-	const allMonths = flatten(
-		map( data, ( year, yearNumber ) => {
-			return map( year, ( month, monthIndex ) => {
-				// keep track of earliest date to fill in zeros when applicable
-				const momentMonth = momentFromMonthYear( monthIndex, yearNumber );
-				if ( momentMonth.isBefore( earliestDate ) ) {
-					earliestDate = moment( momentMonth );
-				}
-				return month[ dataKey ];
-			} );
+	const allMonths = Object.entries( data || {} ).flatMap( ( [ yearNumber, year ] ) =>
```

Comment:
Nitpick: would it be worth doing `data = data || {};` once for the whole method? It seems to always be used with the `data || {}` guard throughout.

Reply:
Did that in 5da822faef, extracting the entire `Object.entries( data )` value to a common variable.

Reply:
Even better, thank you! 👍